resources/views/layouts/master.blade.php

```diff
@@ -24,6 +24,9 @@
     {{--Bootstrap--}}
     <link rel="stylesheet" href="{{ URL::asset('/css/bootstrap.min.css') }}">
 
+    <!-- Jquery -->
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
+
     <style>
```

Comment:
JQuery 3.4 is already included at the bottom of this file... do we need 3.5 for some of the stuff you made?